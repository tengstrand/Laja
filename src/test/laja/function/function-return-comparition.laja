
#function isLessThan(x,y)
  #return x < y
#end

#write "target/test-output/function-return-comparition.out"
  #set (result = isLessThan(1, 2))
  1 is less than 2: {result}
#end
