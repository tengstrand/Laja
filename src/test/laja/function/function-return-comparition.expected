  1 is less than 2: true
